### PR TITLE
Added support for TP-LINK T1600G-52TS

### PR DESCRIPTION
**Pull request by spencerbutler**
This fixes:
#9896

The MIBs (see below) from this vendor don't seem to add up for the T1600G-52TS model. The tpSysMonitorCpuTable Table is what we want to read from, but is not available. I'll see if I can coerce this device into sharing it's CPU and Memory details, but I don't have high hopes at the moment. The device is now detected as "TP-Link JetStream" with this commit.  

T1600G-52TS MIBs
[V1](https://www.tp-link.com/us/download/T1600G-52TS_V1.html) 
[V3](https://www.tp-link.com/us/download/T1600G-52TS_V3.html)
 
TPLINK-SYSMONITOR-MIB::tpSysMonitorCpuTable = No more variables left in this MIB View (It is past the end of the MIB tree)

In V3 they also introduced some new OIDs into the TPLINK-PRODUCTS-MIB Module that are not defined anywhere (l2manageswitch and l3manageswitch), while this isn't germane to this issue, I wanted to mention it.
	tplink-t1600g-52ts	OBJECT IDENTIFIER ::= { tplinkProducts 29 }
	tlsl5210_v1	        OBJECT IDENTIFIER ::= { l3manageswitch 65 }

### vendor MIB
Bad operator (_): At line 48 in mibs/tplink/TPLINK-PRODUCTS-MIB
l3manageswitch.65: Unknown Object Identifier (Sub-id not found: (top) -> l3manageswitch)

Bad operator (_): At line 48 in mibs/tplink/TPLINK-PRODUCTS-MIB
TPLINK-MIB::tplinkProducts.29 = No more variables left in this MIB View (It is past the end of the MIB tree)

### "commented bad OIDs" MIB
l3manageswitch.65: Unknown Object Identifier (Sub-id not found: (top) -> l3manageswitch)

TPLINK-PRODUCTS-MIB::tplink-t1600g-52ts = No more variables left in this MIB View (It is past the end of the MIB tree)

DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [ ] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

**Comment by murrant** [comment]
I don't think the comments in the yaml are needed.  People can look here for extra info if needed.  Please remove them.

Sigh, gotta love those totally broken mibs.

**Comment by spencerbutler** [comment]
> I don't think the comments in the yaml are needed. People can look here for extra info if needed. Please remove them.
> 
> Sigh, gotta love those totally broken mibs.

Yeah, I had no intention of leaving them there. They are more for personal notes while I work on the issue. I'll kill them the next time I checkout this branch.

**Comment by spencerbutler** [comment]
> I don't think the comments in the yaml are needed. People can look here for extra info if needed. Please remove them.
> 
> Sigh, gotta love those totally broken mibs.

commit d65859a kills these

**Comment by spencerbutler** [comment]
Without working MIB files there isn't much more I can do here. If you find working MIBs, please pass them on.

File: includes/definitions/discovery/jetstream.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-mib: TPLINK-SYSMONITOR-MIB
+mib: TPLINK-MIB:TPLINK-PRODUCTS-MIB:TPLINK-SYSMONITOR-MIB
```
Comment (murrant) [inline includes/definitions/discovery/jetstream.yaml]: Since you aren't adding more sensors, are these extra mibs needed?

Comment (spencerbutler) [reply includes/definitions/discovery/jetstream.yaml]: I would like to add sensors, but without working MIBs to work from, I'm stuck.

Comment (murrant) [reply includes/definitions/discovery/jetstream.yaml]: It's fine to add the mibs, but there is no need to import them yet.  If you are yet going to add sensors here, go ahead.  I'm not sure how much those particular mibs will help ;)

**Comment by spencerbutler** [comment]
@SirMaple Can you give this a try and see if it works for you?

**Comment by SirMaple** [comment]
@spencerbutler no change. Ran a manual discovery and poller after applying and still no CPU or Memory graphs for the T1600G-52TS.

**Comment by spencerbutler** [comment]
> @spencerbutler no change. Ran a manual discovery and poller after applying and still no CPU or Memory graphs for the T1600G-52TS.

What OS does it report as?

**Comment by PipoCanaja** [comment]
@SirMaple Did you remove cache/os_defs.cache file before discovery ?

**Comment by SirMaple** [comment]
@spencerbutler I deleted the os_defs.cache as suggested by @PipoCanaja and it works! 

OS reported is now "TP-Link JetStream" the same as the T1600G-28TS.

The 52TS is now also graphing CPU and RAM usage.

Thank you so much!

**Pull request merged**